coursier.bzl

```diff
@@ -119,6 +127,12 @@ def _relativize_and_symlink_file(repository_ctx, absolute_path):
         repository_ctx.symlink(absolute_path, repository_ctx.path(artifact_relative_path))
     return artifact_relative_path
 
+def _get_aar_import_statement_or_empty_str(repository_ctx):
```

Comment:
Is there a way to prevent folks from using this attribute to inject arbitrary strings other than what it should be (a load stmt)? Maybe check for rules_android, aar_import, and newlines?

Reply:
How about using the Label() constructor to try to parse the label. If it's not a valid label, an error should be raised

Reply:
Yep, sounds like a reasonable guardrail.

Reply:
Done.